.storybook/components/Playground/Playground.tsx

```diff
@@ -99,13 +103,12 @@ function getSourceCode(
 }
 
 function getImportStrings(parameters: Story["parameters"]): string {
-  if (parameters && "code" in parameters && parameters.code?.imports) {
-    return parameters.code.imports;
-  }
```

Comment:
Need to update the message here 
![image](https://github.com/GetJobber/atlantis/assets/15986172/409dde1e-2160-47fe-ba12-8e2f76680422)